theme.js

```diff
@@ -0,0 +1,40 @@
+import { slug } from 'github-slugger'
+import path from 'node:path'
+import {
+  MarkdownTheme,
+  MarkdownThemeRenderContext
+} from 'typedoc-plugin-markdown'
+
+const externalLinkRegex = /^(http|ftp)s?:\/\//
+
+export function load(app) {
+  app.renderer.defineTheme('custom-markdown-theme', CustomMarkdownTheme)
+}
+
+export class CustomMarkdownTheme extends MarkdownTheme {
+  getRenderContext(pageEvent) {
+    return new CustomMarkdownThemeContext(pageEvent, this.application.options)
+  }
+}
+
+class CustomMarkdownThemeContext extends MarkdownThemeRenderContext {
+  relativeURL = url => {
+    if (!url) {
+      return null
+    } else if (externalLinkRegex.test(url)) {
+      return url
+    }
+
+    let filePath = path.parse(url)
```

Comment:
Will `parse` works on Windows where we have diffrerent `/`?

Reply:
Good point.

Nope, on Windows there will be broken links.

Reply:
Fixed.